plugins/omprog/omprog.c

```diff
@@ -632,7 +549,7 @@ startChild(wrkrInstanceData_t *pWrkrData)
 {
 	DEFiRet;
 
-	assert(pWrkrData->bIsRunning == 0);
+	assert(!pWrkrData->bIsRunning);
```

Comment:
I admit it's a style issue, so I would not object if you leave as is. I have often seen folks wonder on the abbreviated case, but no strong feelings here. I guess it's done inconsistently in the codebase anyhow ;-)

Reply:
I made this change mostly because the use was inconsistent within omprog. I personally prefer the direct style since it is a boolean (and this is additionally reinforced by the use of the hungarian notation in this case). But I will also be happy with the other style if you prefer.

Reply:
let's keep as you did it - it's inconsistent in any way (and I admit my personal pref is towards this, too, but...)